src/components/molecules/OcTable/OcTable.vue

```diff
@@ -248,8 +255,34 @@ export default {
     },
   },
   methods: {
-    dragStart(file) {
+    dragOver(event) {
+      event.preventDefault()
+    },
+    setGhostElement(file, event) {
+      const selection = [...this.selection]
+      selection.splice(
+        selection.findIndex(function (i) {
+          return i.id === file.id
+        }),
+        1
+      )
+      const GhostElementComponent = Vue.extend(GhostElement)
+      const ghostInstances = new GhostElementComponent({
+        propsData: {
+          previewItems: [file, ...selection],
+        },
+      })
+      ghostInstances.$mount()
+      const ghost = document.body.appendChild(ghostInstances.$el)
```

Comment:
Do you think we need to add `aria-hidden="true"` to the ghost element? It's not possible to initiate it via keyboard, so I'm not sure whether or not we need it. But on the other hand probably doesn't hurt either... what do you think?

Reply:
yes doesnt hurt, added it